src/graph_compute/compute.cpp

```diff
@@ -12,46 +12,69 @@
 #include "./common.h"
 #include "./io.h"
 #include "./update_function.h"
+#include "./concurrent_queue.h"
 
 using namespace std;
 
-#if BASELINE || D1_PRIO
-  #include "./priority_scheduling.h"
+#if BASELINE || D0_BSP
```

Comment:
I don't think it's all that confusing.  Admittedly, it isn't all that necessary, either.  I did it anyway.  Probably not the most unnecessary or confusing change, actually.